x-pack/test/api_integration/deployment_agnostic/default_configs/serverless.config.base.ts

```diff
@@ -70,7 +68,6 @@ export function createServerlessTestConfig(options: CreateTestConfigOptions) {
         serverArgs: [
           ...svlSharedConfig.get('esTestCluster.serverArgs'),
           ...esServerArgsFromController[options.serverlessProject],
-          ...(options.esServerArgs ?? []),
```

Comment:
__note/issue:__ I'm afraid that silently dropping arguments could lead to confusion and hours of debugging when developers try to use the arguments in their configs, and the environment doesn't behave as they expect. Could we instead throw an exception with a clear message if non-empty `esServerArgs` or `kbnServerArgs` are provided?

Reply:
```
*[ftr/update-docs-for-depl-agnostic-tests][~/github/kibana]$ node scripts/functional_tests.js --config x-pack/test/api_integration/deployment_agnostic/search.serverless.config.ts
 warn ❗️❗️❗️
 warn ❗️❗️❗️
 warn ❗️❗️❗️
 warn    Don't forget to use `node scripts/build_kibana_platform_plugins` to build plugins you plan on testing
 warn ❗️❗️❗️
 warn ❗️❗️❗️
 warn ❗️❗️❗️
ERROR UNHANDLED ERROR
ERROR Error: FTR doesn't provision custom ES/Kibana server arguments into the serverless project on MKI.
        It may lead to unexpected test failures on Cloud. Please contact #appex-qa
          at Object.provider (serverless.config.base.ts:59:13)
          at executeConfigModule (config_loading.ts:121:41)
          at readConfigFile (config_loading.ts:169:13)
          at run_tests.ts:70:22
          at tooling_log.ts:84:18
          at runTests (run_tests.ts:64:5)
          at description (cli.ts:24:7)
          at run.ts:73:10
          at withProcRunner (with_proc_runner.ts:29:5)
          at run (run.ts:71:5)
```

Reply:
Niiiice, thanks!